topics/csharp/index.md

```diff
@@ -1,13 +1,16 @@
 ---
-aliases: c-sharp, csharp-code, csharp7, csharp-core, csharp6, csharp-resources
+aliases: c-sharp
 created_by: Anders Hejlsberg
 display_name: C#
+github_url: https://github.com/dotnet/csharplang
 logo: csharp.png
 related: language, dotnet
-released: June 2000
-short_description: C# (C sharp) is an object-oriented programming language by Microsoft.
+released: January 2002
+short_description: C# ("C sharp") is an object-oriented and type-safe programming language.
 topic: csharp
-url: https://docs.microsoft.com/en-us/dotnet/csharp/csharp
+url: https://docs.microsoft.com/dotnet/csharp/
 wikipedia_url: https://en.wikipedia.org/wiki/C_Sharp_(programming_language)
 ---
-C# is a programming language used for web development. It is a hybrid of C and C++ that functions similar to Java and is meant for Microsoft's .NET framework.
+C# (pronounced "C sharp") is a simple, modern, object-oriented, and type-safe programming language.
+
+Its roots in the C family of languages makes C# immediately familiar to C, C++, Java, and JavaScript programmers.
```

Comment:
Should `JavaScript` be included in this list? I consider myself pretty familiar with JavaScript, but I wouldn't say the same for C#. I might even say the same for `Java`. I could be way off base, what are your thoughts?

Reply:
C# and JavaScript are not much related.

Reply:
The text was simply taken from https://dotnet.microsoft.com/languages.

Reply:
If that's the case then I would suggest changing it to something like this:

```suggestion
Its roots in the C family of languages makes C# immediately familiar to C and C++ programmers.
```
I don't think anyone could argue with that assertion.

Reply:
Personally I think JavaScript can be removed. But C# is very similar to Java.

Reply:
/cc @leecow